gradle.properties

```diff
@@ -1,6 +1,5 @@
 # Project Settings
-dokka_version_base=1.6.10
-dokka_publication_channels=maven-central-snapshot&space-dokka-dev
+dokka_version=1.6.20-SNAPSHOT
```

Comment:
Had to add this because integration tests fail otherwise. Previously, it added `-SNAPSHOT` automatically and integration test projects used the SNAPSHOT version when running the tests.

If I revert it back to `1.6.10`, integration tests will use `1.6.10` from maven central, which doesn't have some features yet (like documentedVisibilities), and so the integration tests will fail.

I set it to `1.6.20-SNAPSHOT` specifically since this is the version now in master. I'll change it to `1.6.30-SNAPSHOT` after creating the `1.6.20` release branch.

Reply:
Is it possible to change the integration test to use a special version like [here](https://github.com/Kotlin/dokka/blob/78f25296ad967cba3cfd5bc28a3fd4f5950f5ca0/integration-tests/gradle/projects/template.settings.gradle.kts#L17)?